src/NumericalAlgorithms/Interpolation/Events/InterpolateWithoutInterpComponent.hpp

```diff
@@ -0,0 +1,189 @@
+#include "Domain/ElementId.hpp"
+#include "NumericalAlgorithms/Interpolation/Actions/InterpolationTargetVarsFromElement.hpp"
+#include "NumericalAlgorithms/Interpolation/IrregularInterpolant.hpp"
+#include "NumericalAlgorithms/Interpolation/PointInfoTag.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Parallel/ConstGlobalCache.hpp"
+#include "Parallel/Invoke.hpp"
+#include "ParallelAlgorithms/EventsAndTriggers/Event.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Time/TimeStepId.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace Tags {
+struct TimeStepId;
+}  // namespace Tags
+namespace domain {
+namespace Tags {
+template <size_t VolumeDim>
+struct Mesh;
+}  // namespace Tags
+}  // namespace domain
+namespace Tags {
+template <typename TagsList>
+struct Variables;
+}  // namespace Tags
+template <size_t Dim>
+class Mesh;
+template <size_t VolumeDim>
+class ElementId;
+namespace intrp {
+template <typename Metavariables, typename Tag>
+struct InterpolationTarget;
+}  // namespace intrp
+/// \endcond
+
+namespace intrp {
+namespace Events {
+template <size_t VolumeDim, typename InterpolationTargetTag,
+          typename Metavariables, typename Tensors, typename EventRegistrars>
+class InterpolateWithoutInterpComponent;
+
+namespace Registrars {
+template <size_t VolumeDim, typename InterpolationTargetTag,
+          typename Metavariables, typename Tensors>
+struct InterpolateWithoutInterpComponent {
+  template <typename RegistrarList>
+  using f = Events::InterpolateWithoutInterpComponent<
+      VolumeDim, InterpolationTargetTag, Metavariables, Tensors, RegistrarList>;
+};
+}  // namespace Registrars
+
+/// Does an interpolation onto an InterpolationTargetTag by calling Actions on
+/// the InterpolationTarget component.
+template <size_t VolumeDim, typename InterpolationTargetTag,
+          typename Metavariables, typename Tensors,
+          typename EventRegistrars =
+              tmpl::list<Registrars::InterpolateWithoutInterpComponent<
+                  VolumeDim, InterpolationTargetTag, Metavariables, Tensors>>>
+class InterpolateWithoutInterpComponent;  // IWYU pragma: keep
+
+template <size_t VolumeDim, typename InterpolationTargetTag,
+          typename Metavariables, typename... Tensors, typename EventRegistrars>
+class InterpolateWithoutInterpComponent<VolumeDim, InterpolationTargetTag,
+                                        Metavariables, tmpl::list<Tensors...>,
+                                        EventRegistrars>
+    : public Event<EventRegistrars> {
+  /// \cond
+  explicit InterpolateWithoutInterpComponent(
+      CkMigrateMessage* /*unused*/) noexcept {}
+  using PUP::able::register_constructor;
+  WRAPPED_PUPable_decl_template(InterpolateWithoutInterpComponent);  // NOLINT
+  /// \endcond
+
+  using options = tmpl::list<>;
+  static constexpr OptionString help =
+      "Starts interpolation onto the given InterpolationTargetTag.";
```

Comment:
what do you mean by "Starts" ?  Note that help strings no longer have a character limit, and they are auto-formatted...